### PR TITLE
[DOCS] Will/docs/how to/metadata stores/azure blob for expectations and validations

### PR DESCRIPTION
**Pull request by Shinnnyshinshin**
Changes proposed in this pull request:
- Azure backend for storing Expectations and Validations has not been implemented yet. Added documentation describing this, as well as a link to the Contributor's guide and Slack. 
- 
-

### CODE REVIEW

File: docs/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_on_a_filesystem.rst [hunk 1]
```diff
@@ -3,16 +3,48 @@
...
-.. admonition:: Admonition from Mr. Dickens
 
-    "Whether I shall turn out to be the hero of my own life, or whether that station will be held by anybody else, these pages must show."
+By default, newly profiled Expectations are stored in JSON format in the ``expectations`` subdirectory of your ``great_expectations`` folder.  This guide will help you configure the location of the Expectation store in the filesystem.
 
+.. admonition:: Prerequisites: This how-to guide assumes that you have already:
 
-This guide is a stub. We all know that it will be useful, but no one has made time to write it yet.
+    - Configured a Data Context
+    - Configured an Expectation Suite
```
Comment (jcampbell) [inline docs/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_on_a_filesystem.rst]: Add links to the relevant guides.

File: docs/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_on_a_filesystem.rst [hunk 2]
```diff
@@ -3,16 +3,48 @@
...
+    expectations_store_name: expectations_store
+
+    stores:
+        expectations_store:
+            class_name: ExpectationsStore
+            store_backend:
+                class_name: TupleFilesystemStoreBackend
+                base_directory: expectations/
+
+    The configuration file tells ``Great Expectations`` to look for Expectations in a ``store`` called ``expectation_store``. Further down, the ``base_directory`` for ``expectations_store`` is set to ``expectations/``, which is the default.
```
Comment (jcampbell) [inline docs/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_on_a_filesystem.rst]: This line is getting included in the code block...I think it just needs to be dedented. Also, typo: expectation_store -> expectations_store

File: docs/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_on_a_filesystem.rst [hunk 3]
```diff
@@ -3,16 +3,48 @@
...
+
+2. The following changes will change it to ``new_expectations_store`` with the ``base_directory`` set to ``new_expectations_location``
```
Comment (jcampbell) [inline docs/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_on_a_filesystem.rst]:
```suggestion
2. Update your configuration following the example below. This example would change the Store name to ``new_expectations_store`` with the ``base_directory`` set to ``new_expectations_location``.

    Paths are relative to the directory where ``great_expectations.yml`` is stored.
```

**Review by jcampbell — CHANGES_REQUESTED** [review]

File: docs/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_on_a_filesystem.rst (hunk 1 above)
Comment (Shinnnyshinshin) [reply docs/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_on_a_filesystem.rst]: changed in #1530

File: docs/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_on_a_filesystem.rst (hunk 2 above)
Comment (Shinnnyshinshin) [reply docs/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_on_a_filesystem.rst]: changed in #1530

**Comment by Shinnnyshinshin** [comment]
I reverted `how_to_configure_an_expectation_store_on_a_filesystem.rst` back to what was on the `develop` branch.  It is being modified by PR #1530

**Review by jcampbell — APPROVED** [review]

**Pull request merged**